access-management/manage-private-channels.md

```diff
@@ -0,0 +1,55 @@
+---
+description: This article provides information on managing who has access or is an owner of private channels in Microsoft Teams & Groups.
+---
+
+# Manage Private Channels Access
+
+This article will help set up and manage who the owners and members are for private channels in Microsoft Teams and Groups. 
+
+There are several ways to manage and change the access to private channels from within SysKit Point.
+
+## Manage Access from Microsoft Teams & Groups
+
+* **Click the Microsoft Teams & Groups tile (1)**; this is located on the home screen of SysKit Point. 
+* **Select the team or group to manage (2)**; clicking the selected resource opens the details screen.
+* On the details screen, **scroll to the Channels section (3)**; the private teams and groups will be marked by a locked symbol next to their name. 
+* **Click the private channel (4)**; this opens the details screen of the channel.
+
+![Private Channels Access - Microsoft Teams & Groups](../.gitbook/assets/manage-private-channels_teams-and-groups-access.png)
+
+On the left side of the screen, there are two options available under Manage:
+ * **Add Owners/Members (1)**; the pop-up that opens allows adding new members to the private channel. Once added, these members can be promoted to Owner.
+ * **Change Owners (2)**; the pop-up that opens provides the ability to remove current owners or promote current members to owners. 
+
+ ![Manage Private Channels - Options Panel](../.gitbook/assets/manage-private-channels_options-side.png)
+
+## Manage Access from Sites
```

Comment:
ovo bih možda samo spomenuo kao alternativni način kako se dođe do PC details screena, ali ne čini mmi se kao da zahtijeva posebnu sekciju